workflows/crisprseq_screening.nf

```diff
@@ -145,15 +154,52 @@ workflow CRISPRSEQ_SCREENING {
 
     if(params.rra_contrasts) {
         Channel.fromPath(params.rra_contrasts)
-            .splitCsv(header:true, sep:',' )
+            .splitCsv(header:true, sep:';' )
             .set { ch_contrasts }
         counts = ch_contrasts.combine(ch_counts)
 
         MAGECK_TEST (
             counts
         )
+
+        //MAGECK_TEST.out.gene_summary.dump(tag:"test")
+        MAGECK_GRAPHRRA (
+            MAGECK_TEST.out.gene_summary
+        )
     }
 
+    if(params.rra_contrasts) {
+        Channel.fromPath(params.rra_contrasts)
+            .splitCsv(header:true, sep:';' )
+            .set { ch_bagel }
+    counts = ch_bagel.combine(ch_counts)
```

Comment:
Have you tested what happens if `params.rra_contrasts` is false and `ch_bagel` is never created? Not sure if it will fail

Reply:
ah ok! I didn't see it was inside the if clause :)